website_blog_mail_server/models/mail_message.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class MailMessage(models.Model):
+    _inherit = "mail.message"
+
+    @api.model
+    def _get_reply_to(self, values):
```

Comment:
@TimLai125 Can you please explain why we should override the reply-to address here. I wasn't sure.